src/mistral_common/tokens/tokenizers/instruct.py

```diff
@@ -995,6 +1015,20 @@ class InstructTokenizerV13(InstructTokenizerV11):
 
     _user_message_position_to_encode_tools = UserMessagePosition.first
 
+    def __init__(
+        self,
+        tokenizer: Tokenizer,
+        image_encoder: Optional[ImageEncoder] = None,
+        audio_encoder: Optional[AudioEncoder] = None,
+    ) -> None:
+        super().__init__(tokenizer, image_encoder, audio_encoder)
+        try:
```

Comment:
no try ... except here please. Better to instead make use of private attribute: `tokenizer._all_special_tokens`